web/src/main/java/se/inera/intyg/intygstjanst/web/service/CitizenCertificateTextService.java

```diff
@@ -0,0 +1,9 @@
+package se.inera.intyg.intygstjanst.web.service;
+
+import se.inera.intyg.common.support.modules.registry.ModuleNotFoundException;
+
+public interface CitizenCertificateTextService {
```

Comment:
Avoid throwing ModuleNotFoundException

Reply:
How should I handle the error instead?

Reply:
Catch it and throw a RuntimeException with the original exception. Then the code that uses this service doesn't have to bother about it. If this error occurs, we don't really have a way to recover and better let it "crash" fast.